container_pipeline/scanners/misc_package_updates.py

```diff
@@ -59,7 +59,9 @@ def process_output(self, logs):
         """
         data = {}
         data["scanner_name"] = self.scanner_name
-        data["msg"] = "Results for miscellaneous package manager updates"
+        data["msg"] = ""
+        for i in logs:
+            data["msg"] += i["Summary"]
```

Comment:
here do we say we would be having summary for each of the package management we support?

Reply:
@bamachrn didn't get you. In the email output that I've mentioned in first comment, the line `Possible updates for packages installed via pip. Possible updates for packages installed via npm. Possible updates for packages installed via gem.` that you see is a result of string concatenation of `i["Summary"]` for `pip`, `npm` and `gem` scanner results.

Reply:
got it